src/taipy/gui/gui_actions.py

```diff
@@ -243,6 +243,29 @@ def invoke_callback(
     _warn("'invoke_callback()' must be called with a valid Gui instance.")
 
 
+def invoke_shared_callback(
+    gui: Gui,
+    callback: t.Callable,
+    args: t.Union[t.Tuple, t.List],
+    module_context: t.Optional[str] = None,
+) -> t.Any:
+    """Invoke a user callback using state with shared variables.
+
+    See the [User Manual section on Long Running Callbacks in a Thread](../../gui/callbacks/#long-running-callbacks-in-a-thread)
```

Comment:
long running ?

Reply:
@FabienLelaquais can verify I guess

Reply:
Yes. I would actually replace the doc by "TODO".
So we don't forget.

Fred points out that the current content refers to the long_callback thing. Which is wrong.